dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/UserSettingController.java

```diff
@@ -119,7 +119,7 @@ public Map<String, Serializable> getAllUserSettings(
         return result;
     }
 
-    @GetMapping( value = "/{key}" )
+    @GetMapping( value = "/{key}", produces = "text/plain" )
```

Comment:
FYI: this is just making clear what the actual behaviour always has been. This method does not produce JSON as the returned string is not quoted.